app/controllers/Checkout.scala

```diff
@@ -157,6 +136,7 @@ class Checkout(fBackendFactory: TouchpointBackends, commonActions: CommonActions
                 getSettings(determinedCountryGroup.defaultCountry, determinedCountryGroup.currency)
               }
             }
+            case _ => getSettings(determinedCountryGroup.defaultCountry, determinedCountryGroup.currency)
```

Comment:
This seemed like a sensible default case to me

Reply:
👍 agreed